### ISSUE TITLE
Anonymous Edit link is dead - cookies were preserved

### ISSUE THREAD

**Issue by rfprogram** [umap-project/umap#1946]
No other users have my edit link, just me. 
As of last week, the edit link was functional. 
Cross-confirmed, the edits made then are still intact. 
And at that time, I made a point to preserve the umap cookies in my browser (Firefox). 
Confirmed we do have cookies from openstreetmap, umap, and openstreetmap.fr but we did reload a lot so FF is only showing the updates within the last hour. 
However, since then Firefox updated, and I don't know if that makes a difference. But even if I messed up and cleared cookies, it was on the white-list. Nothing from the above websites should have been deleted. 
There is no error message at all for the edit link. It's just a blank page. I switched VPN addresses in case we were on a blacklist there. Same result. 

https://umap.openstreetmap.fr/en/map/konoha-analogues_967855

**Comment by yohanboniface** [comment]
Thanks for the report. We're having a look at it!

**Comment by yohanboniface** [comment]
Should be fixed, can you confirm ?

**Comment by rfprogram** [comment]
yep! thank you so much. sorry for the delay, had to find the stuff to test it with. looks in order!
